tests/spec/health_spec.lua

```
@@ -0,0 +1,280 @@
+-- Tests for caramba.health module
+-- Comprehensive test suite covering health checks and system validation
+
+-- Mock vim API for testing
+local mock_vim = {
+  health = {
+    start = function(name) 
```

Luacheck (1.2.0)

[warning] line contains trailing whitespace

(W612)